### ISSUE TITLE
What is the purpose of this repo?

### ISSUE THREAD

**Issue by cornfeedhobo** [grafana/k8s-monitoring-helm#310]
I'm investigating grafana/helm-charts#1397 and stumbled on this repo. Can someone explain the purposed of this repo vs `helm-charts`?

**Comment by petewall** [comment]
Hello! Great question!
In short, the `helm-charts` repo is for two things:
1. Community-driven charts and some Grafana Labs-driven charts.
2. GitHub releases hosts the Helm Chart repository (for `helm repo ...` commands)

When we started making this chart, we long debated if it would belong in that git repo, or in its own git repo (where it is now).

We decided on a separate repo because it gave us a few benefits:
1. The Grafana Labs team working on [Kubernetes Monitoring on Grafana Cloud](https://grafana.com/solutions/kubernetes/) can have more hands-on control over the direction and curation of the configuration deployed by this chart.
2. It allow us easier control of CI/CD integrations, like making specific GitHub actions for testing, and maintaining this repo.
